### ISSUE TITLE
Add support for Mapzen terrain

### ISSUE THREAD

**Issue by mourner** [mapbox/mapbox-gl-native#11204]
Native needs a port of https://github.com/mapbox/mapbox-gl-js/pull/6110. Will add the render test to ignores for now. cc @mollymerp

**Comment by mourner** [comment]
Oh, it looks like ignores won't do because CI Nitpick complains about shaders/docs not matching. Let's wait on https://github.com/mapbox/mapbox-gl-native/pull/11154 merge then before merging heatmaps.

**Comment by mollymerp** [comment]
@mourner PR for mapzen is here #11154
